### ISSUE TITLE
Theme select

### ISSUE THREAD

**Issue by cb3hn** [petrucci4prez/CinnXP#4]
I get an error in Linux Mint 17.3 Cinnamon in the systemsettings/Themes during over online Themes install:

Details: [Errno 2] Datei oder Verzeichnis nicht gefunden: u'/tmp/tmpISqYSX/CinnXP/cinnamon/cinnamon.css'

**Comment by ndwarshuis** [comment]
did you try manually installing the theme?

This bug happened because of the way the package is structured. There are "theme" files (gtk, metacity, and cinnamon) which all go to ~/.themes or /usr/share/themes, and there are "icon" files (which for now only includes the cursor theme, but will soon include a native icon theme...hopefully) which belong in ~/.icons or /usr/share/icons. As far as I can tell, the mint auto-installer will simply take the package on spices and dump its contents into ~/.themes, regardless of whether or not it has an icon component (probably because it's really only supposed to be used for cinnamon themes)

tl dr
this theme cannot be fully installed within mint itself; it doesn't know how to handle each component. I would prefer to leave the file structure as is to discourage people from only installing part of the theme

**Comment by NikoKrause** [comment]
@petrucci4prez 
What are your plans for this theme in Linux Mint 18? Are you willing to get it work there? If yes: How much time (approximately) do you thing, you will need to make it work in LM18?

**Comment by ndwarshuis** [comment]
I'm not anticipating LM18 to be a huge upgrade. According to clem its supposed to use gtk3.18 so I would guess that 90% of this theme will work as is...the remaining 10% will be the cinnamon theme (which usually isn't huge) and special cases where the LM team is trying to avoid CSD with their own forked apps

short answer: yes, probably a few weeks to a month after LM18 release

**Comment by NikoKrause** [comment]
@petrucci4prez 
It's great, that you added your theme for Cinnamon 3.0 so quickly.

I have an idea, but don't know, if it's possible:
The sidebar in Win XPs file explorer has pale blue boxes. Do you think this could be done in nemo?
(I made (bad) picture in Gimp, to make it more clear, what I mean.) 

![39d36c66-bf65-11e5-9a72-5065f4fa1e72](https://cloud.githubusercontent.com/assets/8415124/15277075/e42be5ae-1af9-11e6-878f-faa4031e1711.png)

**Comment by ndwarshuis** [comment]
not really, I can make a blue border with light blue in the center but the padding/spacing will look really bad

**Comment by NikoKrause** [comment]
Okay, never mind. It was just an idea. Now I know, that it's not possible in nemo.

**Comment by cinnamoner** [comment]
@NikoKrause 
You certainly know what you want. Thanks for the effort you are putting into the thing.

**Comment by Elbullazul** [comment]
@nikokrause actually you can get something similar to it,  but it wont look like the real thing. 

I'll post the code later

**Comment by NikoKrause** [comment]
@Elbullazul 
Thanks. I'm curious what you have in mind.

**Comment by Elbullazul** [comment]
@NikoKrause here you go : 
![capture du 2016-06-09 09 59 39](https://cloud.githubusercontent.com/assets/15310985/15932353/58092076-2e29-11e6-8840-885766e1bdc0.png)

to achieve this, search for 

> NemoWindow .sidebar .frame {

and replace border-width: 0; with

> border: 4px solid #d6dff7;

that should do it

or, if you prefer, set the sidebar bg color to #d6dff7 and the border color to #687FD9

**Comment by NikoKrause** [comment]
@Elbullazul 
Sorry, but I don't get, what it does?

**Comment by Elbullazul** [comment]
kind of a border inside the column

the other alternative would be kind of something with the row elements, but I can't get it to work yet

**Comment by NikoKrause** [comment]
@Elbullazul 
I'll try it later though. In which file do I find NemoWindows?

**Comment by Elbullazul** [comment]
gtk-widgets.css

**Comment by cinnamoner** [comment]
@Elbullazul 
Yeah. It seems to look better that way.

**Comment by dominichayesferen** [comment]
@cinnamoner Exactly what I thought. The only problem would be trying to come up with an XP-like Focus square colour for the selected box around the location in the pane...

Also, I'd flip the colours around so the outside is dark, while the inside it light

**Comment by dominichayesferen** [comment]
@Elbullazul No offense, but if you're deprecating XP themes, then why are they still alive on your website?

**Comment by Elbullazul** [comment]
@feren I'm just feeling lazy lately... :/

**Comment by NikoKrause** [comment]
@petrucci4prez 
To let you know:
> In ~/.icons, make symbolic link called "default" and point it to the CinnXP cursor directory (this is required for QT apps and probably other stuff)

This issue with the cursor theme not changing for QT apps has been fixed in Cinnamon 3.4.6.
Now when you change the cursor theme to e.g. CinnXP cursors, a folder `default` will be created in `~/.icons` with a file `index.theme`:
```
[icon theme]
Inherits=CinnXP
```

So creating a symbolic link manually is no longer needed.
